data-prepper-main/src/main/java/org/opensearch/dataprepper/DataPrepperExecute.java

```diff
@@ -19,7 +19,7 @@ public class DataPrepperExecute {
     public static void main(final String ... args) {
         java.security.Security.setProperty("networkaddress.cache.ttl", "60");
 
-        final ContextManager contextManager = new ContextManager(args);
+        final ContextManager contextManager = new ContextManager();
```

Comment:
I think it would be more appropriate to get the root directory here. Then get the config and pipelines values. Then you can pass them into `ContextManager` using the existing constructor.

Basically:

```
String dataPrepperHome = System.getProperty("data-prepper.dir");
String dataPrepperConfig = dataPrepperHome + "/config/data-prepper-config.yaml";
String dataPrepperPipelines = dataPrepperHome + "/pipelines/pipelines.yaml";
final ContextManager contextManager = new ContextManager(dataPrepperPipelines, dataPrepperConfig);
```

Reply:
Also, we should throw a clear exception when `data-prepper.dir` is not set.